src/pages/@owner/food-truck-form/FoodTruckForm.tsx

```
@@ -0,0 +1,110 @@
+  FoodTruckPhoto,
+} from '@pages/@owner/food-truck-form/@section/basic-info-section';
+import {
+  ActiveTime,
+  ActiveDate,
+} from '@pages/@owner/food-truck-form/@section/time-section';
+import {
+  AvailableQuantity,
+  NeedElectricity,
+  PaymentMethod,
+} from './@section/category-section';
+import { MenuCategory } from './@section/category-section';
```

ESLint

[error] './@section/category-section' import is duplicated.

(no-duplicate-imports)